src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -67,6 +68,25 @@ const EventEntries = createReactClass({
     };
   },
 
+  recordSourcemapError(errorType) {
+    analytics('sourcemap.sourcemap_error', {
+      'org_id': this.props.project.organization.id,
+      'group': this.props.event.groupID,
+      'error_type': errorType
+    })
+  },
+
+  componentDidMount() {
+    if (typeof this.props.event.errors !== 'undefined') {
```

Comment:
same deal here with destructuring `let {event} = this.props;`

Reply:
For this check a pattern we use a bunch is: 
`if (!this.props.event.errors) return;`